desktop/apps/editorial_features/components/gucci/templates/head.jade

```diff
@@ -7,6 +7,10 @@ else
 
 title= page.title || page.name
 link( rel="canonical", href=url )
+
+//- Unica Font
+link( rel='stylesheet' type='text/css' href="#{sd.WEBFONT_URL + '/unica-webfonts.css'}")
```

Comment:
Ah 👍  bc the Force one doesn't have Unica -- once more pages start using Unica (I think I saw comps for Artist page), we should add it to the Force sheet for all to use.